api/models/rules/rules.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+
+from api.models.choices.log_event_types import (
+    CoordinatorLogEventType,
+    GroupLogEventType,
+    StudentLogEventType,
+    TeacherLogEventType,
+    TeacherUnder18LogEventType,
+)
+from api.models.choices.statuses import (
+    CoordinatorStatus,
+    GroupStatus,
+    StudentStatus,
+    TeacherStatus,
+    TeacherUnder18Status,
+)
+
+LogEventType = (
```

Comment:
Maybe do some base class for all event types, like Protocol (https://peps.python.org/pep-0544/) ?